### PR TITLE
REVIEW: [NEXUS-3742] Add grace days after release on snapshot remover

### PR DESCRIPTION
**Pull request by adreghiciu**
https://issues.sonatype.org/browse/NEXUS-3742
https://bamboo.zion.sonatype.com/browse/NXO-OSSF5

### CODE REVIEW

File: nexus-core/src/main/java/org/sonatype/nexus/maven/tasks/DefaultSnapshotRemover.java [hunk 1]
```diff
@@ -722,7 +737,9 @@ public boolean releaseExistsForSnapshot( Gav snapshotGav, Map<String, Object> co
                 }
             }
 
-            return false;
+            getLogger().info( "--------- {} {} {} {}", snapshotGav.getName(), new Date(releaseTimestamp), new Date(startTime), new Date(releaseTimestamp + gracePeriodInMillis) );
```
Comment (jdillon) [inline nexus-core/src/main/java/org/sonatype/nexus/maven/tasks/DefaultSnapshotRemover.java]: strange info level logging

Comment (adreghiciu) [inline nexus-core/src/main/java/org/sonatype/nexus/maven/tasks/DefaultSnapshotRemover.java]: That was not supposed to be there ;) Tx.

**Comment by adreghiciu** [comment]
Ci :+1: https://bamboo.zion.sonatype.com/browse/NXO-OSSF5-1

**Comment by kellyrob99** [comment]
+1

**Comment by nabcos** [comment]
+1

**Comment by cstamas** [comment]
+1

**Pull request merged**